### PR TITLE
IMAP module may now check several IMAP folders

### PR DESCRIPTION
**Pull request by rixx**
Returns sum of unseen mails in given folders. Former behaviour with only
one given directory is unchanged.

(I wanted the mail number to reflect a mailing list directory of mine, hence the change)

### CODE REVIEW

File: py3status/modules/imap.py [hunk 1]
```diff
@@ -42,13 +43,19 @@ def check_mail(self, i3s_output_list, i3s_config):
...
-            connection.select(self.mailbox)
-            unseen_response = connection.search(None, self.criterion)
-            mails = unseen_response[1][0].split()
-            mail_count = len(mails)
+            mail_count = 0
+            directories = self.mailbox.split(',')
+            
+            for directory in directories:
+                connection = imaplib.IMAP4_SSL(self.imap_server, self.port)
+                connection.login(self.user, self.password)
```
Comment (ultrabug) [inline py3status/modules/imap.py]: Maybe we could only connect once and login once before the loop (and close after) and instead only select the given directory in the loop ?

**Comment by ultrabug** [comment]
Thanks @rixx, this is indeed a good thing to propose. I permited myself with a comment to save resources on your code, please review ?

**Comment by rixx** [comment]
Yeah, sorry, should have thought of that right away. Worked it in, here you go.

File: py3status/modules/imap.py [hunk 2]
```diff
@@ -42,13 +43,19 @@ def check_mail(self, i3s_output_list, i3s_config):
...
             connection.login(self.user, self.password)
-            connection.select(self.mailbox)
-            unseen_response = connection.search(None, self.criterion)
-            mails = unseen_response[1][0].split()
-            mail_count = len(mails)
+            
+            for directory in directories:
+                connection.select(directory)
+                unseen_response = connection.search(None, self.criterion)
+                connection.close()
```
Comment (ultrabug) [inline py3status/modules/imap.py]: little glitch here and we're done, should be closed outside the `for`, just before the return I guess plz

**Comment by ultrabug** [comment]
Thx @rixx, only one forgotten line as per comment and we're done :)

**Comment by rixx** [comment]
d'oh, sorry, fixed.

**Comment by ultrabug** [comment]
Thanks a lot (again) mate :)

**Pull request merged**